test/base/BaseModel.spec.js

```diff
@@ -4,10 +4,39 @@ import _isString from 'lodash/isString';
 import BaseModel from '../../src/assets/scripts/client/base/BaseModel';
 import ExtendedBaseModelFixture from './_fixtures/ExtendedBaseModelFixture';
 
+ava('sets the #_id when passed valid parameters', (t) => {
+    const modelInt = new BaseModel();
+    t.true(modelInt._id.indexOf('BaseModel') !== -1);
+
+    const modelBool = new BaseModel('string');
+    t.true(modelBool._id.indexOf('string') !== -1);
+    console.log(modelBool._id);
+});
+
+ava('makes sure the model fails to instantiate with any incorrect', t => {
+    t.notThrows(() => new BaseModel());
+    t.throws(() => new BaseModel([]));
+    t.throws(() => new BaseModel({}));
+    t.throws(() => new BaseModel(42));
+    t.notThrows(() => new BaseModel('string'));
+    t.throws(() => new BaseModel(false));
+});
+
+ava('makes sure the model in instantiates with the correct data type', t => {
+    t.notThrows(() => new BaseModel());
+    t.notThrows(() => new BaseModel('string'));
+});
+
 ava('instantiates with an _id property', t => {
-    const result = new BaseModel();
+    const resultEmpty = new BaseModel();
```

Comment:
be sure to use `model` as the const name.  this is just an [undocumented] convention we've established in our tests to make it more readable.